pkg/cloudprovider/providers/aws/aws_routes.go

```diff
@@ -49,6 +49,12 @@ func (s *AWSCloud) findRouteTable(clusterName string) (*ec2.RouteTable, error) {
 
 // ListRoutes implements Routes.ListRoutes
 // List all routes that match the filter
+func getInstanceAndCIDR(r *ec2.Route)(string, string) {
```

Comment:
It just repeats the same code twice in both `for` loops thats why I created it.  I'll be glad to remove it if you think that style is better? what do you think?